contracts/mechs/nevermined/BalanceTrackerNvmSubscriptionNative.sol

```diff
@@ -54,13 +48,19 @@ contract BalanceTrackerNvmSubscription is BalanceTrackerBase {
     // Temporary owner address
     address public owner;
 
-    // TODO Do we manage subscription fee via buyBackBurner as well or in-place?
     /// @dev BalanceTrackerSubscription constructor.
     /// @param _mechMarketplace Mech marketplace address.
     /// @param _buyBackBurner Buy back burner address.
-    constructor(address _mechMarketplace, address _buyBackBurner)
-        BalanceTrackerBase(_mechMarketplace, _buyBackBurner)
+    /// @param _wrappedNativeToken Wrapped native token address.
+    /// @param _creditTokenRatio Credits to token ratio.
+    constructor(address _mechMarketplace, address _buyBackBurner, address _wrappedNativeToken, uint256 _creditTokenRatio)
+        BalanceTrackerFixedPriceNative(_mechMarketplace, _buyBackBurner, _wrappedNativeToken)
```

Comment:
Good news is that buyBackBurner is back in game here, as ultimate withdraws are managed by native / ERC20 tokens.